bodhi/server/mail.py

```diff
@@ -182,7 +182,7 @@
 """,
         'fields': lambda agent, x: {
             'package': x.title,
-            'comment': x.comments[-1],
+            'comment': x.comments[-1].text,
```

Comment:
@ralphbean I had been having some encoding related issues when I was developing this patch early on, and I think I had done this to test that. I reverted this change and all tests pass so I think it's not needed. I'll back it out.